connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorMakerConfig.java

```diff
@@ -245,10 +254,17 @@ public MirrorClientConfig clientConfig(String cluster) {
         Map<String, String> strings = originalsStrings();
         strings.keySet().removeIf(x -> !x.startsWith(prefix));
         return strings;
-    } 
+    }
+
+    static Map<String, String> prefixForGeneralAttrs(String prefix, Map<String, String> props) {
```

Comment:
Can we find a better name for these 2 new methods? Something like `clusterConfigsWithPrefix()` and `clientConfigsWithPrefix()`?

Reply:
Indeed. Thanks!